gremlin-go/driver/gorillaTransporter_test.go

```diff
@@ -30,9 +30,11 @@ import (
 )
 
 const mockMessage string = "MockMessage"
-const mockWriteErrMessage string = "MockWriteJSONErrMessage"
+
+// const mockWriteErrMessage string = "MockWriteJSONErrMessage"
 const mockReadErrMessage string = "MockReadMessageErrMessage"
-const mockCloseErrMessage string = "MockCloseErrMessage"
+
+// const mockCloseErrMessage string = "MockCloseErrMessage"
```

Comment:
staticcheck warned that the ones I commented out were unused, should they be removed?

Reply:
If they are no longer used you can remove them.

Reply:
I think if they're unused you're good to remove them.